docs/developers/QueryTrace.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: page
+title: How To Use Gluten
+nav_order: 1
+parent: Developer Overview
+---
```

Comment:
The header needs to update.

Reply:
@jinchengchenghh 

nit: Would you update `title` (and perhaps other fields as well) in the header? Thanks.